editor/assets/default_renderpipeline/builtin-pipeline-settings.ts

```diff
@@ -195,6 +195,9 @@ export class BuiltinPipelineSettings extends Component {
     })
     set dofMaterial(value: Material) {
         this._settings.depthOfField.material = value;
+        if (EDITOR) {
+            this._tryEnableEditorPreview();
```

Comment:
Should check value before doing it? If the value is not changed, then don't have to do it.

Reply:
Value check is added.